### ISSUE TITLE
multiple querystring params ignored

### ISSUE THREAD

**Issue by jgelens** [eliangcs/http-prompt#20]
Httpie supports sending multiple querystring args with the same key. http-prompt uses a dict behind the scenes and therefore only calls Httpie with a single value:

```
http://localhost:6543/testing> httpie get par==1 par==2
http GET http://localhost:6543/testing par==2
```

This should result in:
```
http://localhost:6543/testing> httpie get par==1 par==2
http GET http://localhost:6543/testing par==1 par==2
```

**Comment by eliangcs** [comment]
I did it on purpose because I was considering the following scenario:

```
par==1 par==2
par==3
httpie
```

The current implementation would output `http http://localhost par==3`. But if I changed the implementation so that it outpus `http http://localhost par==1 par==2 page==3`, how does a user update an existing querystring parameter? I did not spend too much time thinking of a good design for this, so I just assume users wouldn't have multiple querystring parameters with the same name.

**Comment by Zaijo** [comment]
@jgelens When trying to accomplish what you intend I usually type `httpie <method>` or just `httpie` into `http-prompt`, copy the result, add the repeating param and run httpie itself.

**Comment by twz123** [comment]
I also noticed this issue. Since `rm` is there to remove no longer wanted parameters, why not adding a comparable command like `append` to add additional values to an existing parameter? When thinking about it, then `rm` would also need to support the removal of single values from a parameter in addition to the removal of the whole parameter.

**Comment by eliangcs** [comment]
@twz123 thanks for your suggestion, but `append` command doesn't seem beautiful to me. All of the existing commands, such as `cd` and `rm`, are borrowed from Unix commands. If I were to add a new command, I want it to be like a Unix command as well. Until we come up with a satisfying design, I guess you can use the workaround that @Zaijo suggested.

**Comment by eliangcs** [comment]
I think we can follow httpie's design. When we have multiple assignments in a single line, it should create multiple values for that parameter. For instance:

```
> rm *
> par==1 par==2
> httpie
http http://localhost par==1 par==2
```

And if the assignment is done in a separate line, the parameter is overwritten by the last line entirely. For instance:

```
> rm *
> par==1 par==2
> par==3 par==4
> httpie
http http://localhost par==3 par==4
```

@jgelens does this solution work for you?

**Comment by jgelens** [comment]
Sounds great :-)

**Comment by eliangcs** [comment]
Released with v0.4.2. Please upgrade it with:

```
$ pip install -U http-prompt
```
